packages/phrases/src/locales/en.ts

```diff
@@ -1,658 +1,3 @@
-    not_found: 'Cannot find any available connector for type: {{type}}.',
-    not_enabled: 'The connector is not enabled.',
-    insufficient_request_parameters: 'The request might miss some input parameters.',
-    invalid_config: "The connector's config is invalid.",
-    invalid_response: "The connector's response is invalid.",
-    template_not_found: 'Unable to find correct template in connector config.',
-    invalid_access_token: "The connector's access token is invalid.",
-    invalid_auth_code: "The connector's auth code is invalid.",
-    invalid_id_token: "The connector's id token is invalid.",
-    authorization_failed: "The user's authorization process is unsuccessful.",
-    oauth_code_invalid: 'Unable to get access token, please check authorization code.',
-    more_than_one_sms: 'The number of SMS connectors is larger then 1.',
-    more_than_one_email: 'The number of Email connectors is larger then 1.',
-    db_connector_type_mismatch: 'There is a connector in the DB that does not match the type.',
-  },
-  passcode: {
-    phone_email_empty: 'Both phone and email are empty.',
-    not_found: 'Passcode not found. Please send passcode first.',
-    phone_mismatch: 'Phone mismatch. Please request a new passcode.',
-    email_mismatch: 'Email mismatch. Please request a new passcode.',
-    code_mismatch: 'Invalid passcode.',
-    expired: 'Passcode has expired. Please request a new passcode.',
-    exceed_max_try: 'Passcode verification limitation exceeded. Please request a new passcode.',
-  },
-  sign_in_experiences: {
-    empty_content_url_of_terms_of_use:
-      'Empty "Terms of use" content URL. Please add the content URL if "Terms of use" is enabled.',
-    empty_logo: 'Please enter your logo URL',
-    empty_slogan:
-      'Empty branding slogan. Please add a branding slogan if a UI style containing the slogan is selected.',
-    empty_social_connectors:
-      'Empty social connectors. Please add enabled social connectors when the social sign-in method is enabled.',
-    enabled_connector_not_found: 'Enabled {{type}} connector not found.',
-    not_one_and_only_one_primary_sign_in_method:
-      'There must be one and only one primary sign-in method. Please check your input.',
-  },
-  swagger: {
-    invalid_zod_type: 'Invalid Zod type. Please check route guard config.',
-    not_supported_zod_type_for_params:
-      'Not supported Zod type for the parameters. Please check route guard config.',
-  },
-  entity: {
-    create_failed: 'Failed to create {{name}}.',
-    not_exists: 'The {{name}} does not exist.',
-    not_exists_with_id: 'The {{name}} with ID `{{id}}` does not exist.',
-    not_found: 'The resource does not exist.',
-  },
-};
-
-const en = Object.freeze({
-  translation,
-  errors,
-});
-
-export default en;
-/* eslint-enable max-lines */
+// Note: we need to re-export the locale under this `locales` folder,
+// so that the `useTranslation` function can infer the i18n key type.
```

Comment:
1. use multiline comment here
2. record what error will throws if we don't do this

```ts
/**
 * line 1
 * line 2
 */
```